pkg/cacheutil/memcached_client.go

```diff
@@ -158,6 +180,12 @@ func (c *MemcachedClientConfig) validate() error {
 		return errMemcachedMaxAsyncConcurrencyNotPositive
 	}
 
+	if c.SetAsyncCircuitBreakerConsecutiveFailures == 0 {
+		return errCircuitBreakerConsecutiveFailuresNotPositive
+	}
+	if c.SetAsyncCircuitBreakerFailurePercent <= 0 || c.SetAsyncCircuitBreakerFailurePercent > 1 {
+		return errCircuitBreakerFailurePercentInvalid
+	}
```

Comment:
Since we enable the circuit breaker by default, is there any downside to do so? Do you think it is worth it to put it behind a FF?

Reply:
feature flag has been added